src/main/java/net/consensys/orion/config/Config.java

```diff
@@ -245,18 +246,36 @@ public Path workDir() {
    * Storage engine used to save payloads and related information. Options:
    *
    * <ul>
-   * <li>leveldb:path - LevelDB
-   * <li>mapdb:path - MapDB
+   * <li>leveldb:path - LevelDB</li>
+   * <li>mapdb:path - MapDB</li>
    * <li>sql:jdbcurl - Relational database</li>
-   * <li>memory - Contents are cleared when Orion exits
+   * <li>memory - Contents are cleared when Orion exits</li>
    * </ul>
    *
    * <strong>Default:</strong> "leveldb"
    *
    * @return Storage string specifying a storage engine and/or storage path
    */
   public String storage() {
-    return configuration.getString("storage");
+    return System.getenv().getOrDefault("ORION_STORAGE", configuration.getString("storage"));
+  }
+
+  /**
+   * Storage engine used to save node information. Options:
+   *
+   * <ul>
+   * <li>leveldb:path - LevelDB</li>
+   * <li>mapdb:path - MapDB</li>
+   * <li>sql:jdbcurl - Relational database</li>
+   * <li>memory - Contents are cleared when Orion exits</li>
+   * </ul>
+   *
+   * <strong>Default:</strong> "memory"
+   *
+   * @return Storage string specifying a storage engine and/or storage path
+   */
+  public String nodeStorage() {
+    return System.getenv().getOrDefault("ORION_NODE_STORAGE", configuration.getString("nodestorage"));
```

Comment:
Once more, see my comment about this node storage name :)

Reply:
OK done.